src/system/unwind.c

```diff
@@ -114,6 +114,7 @@ _Unwind_Reason_Code trace_fn(_Unwind_Context* unwind_ctx, void* d) {
 /******************************************************************************/
 // recover registers from a data abort. Specific knowledge about callee stack
 // used from FreeRTOS_DataAbortHandler and it calling DataAbortInterrupt
+__attribute((target("arm")))
```

Comment:
Since when can you call `__attribute` without the trailing `__`?

Reply:
Thanks for catching this, fixed in https://github.com/purduesigbots/pros/pull/708/commits/8b9cfae4726ca3191efb34155bf8d3d13538a26c.